content/post/proficiency_v_creativity.md

```diff
@@ -0,0 +1,63 @@
+
+Leading for creativity means fostering systems that consistently churn out fresh ideas, even if many of those ideas don’t turn out to be successful. Empowerment is critical - providing support and tools to enable the team and letting them run with it. Creativity also requires a real focus on the team’s dynamic and sense of psychological safety. Teams should celebrate individuals who take initiative and generate new ideas, but not all those ideas will ultimately be good ones. Folks need to feel safe failing, suggesting something they aren’t quite sure is fully baked, and giving authentic feedback on someone else’s idea in an honest but respectful way.
+
+Tactically, enabling creativity requires time to work on less structured projects and a clear path to pursue ideas. Folks need time for explorations that may not always result in clear outcomes. Sometimes you spend half a day investigating something you think will be meaningful, and it’s just not - or weeks building a model only to find out that you just can’t predict the outcome with any useful level of accuracy. That time wasn’t wasted if you learn something from it.
+
+## The Balance
+The magic of a data team is the combination of rigor and open endedness - it isn’t a pyramid, where we need one to enable the other as an ultimate goal. It’s more complementary and interconnected - the yin of proficiency and the yang of creativity, if you will. Focus too much on proficiency, and you’ll have accurate reports created by people who are probably bored every day and stakeholders who don’t really understand the value that the data team can add. Enable too much creativity, and you’ve got folks making decisions based on questionable data or a team that works on projects that are interesting but not relevant to the business.
+
+Maintaining this balance relies on building a team culture that celebrates both rigor and exploration, but also requires a concerted effort just to make time for creative projects. It’s easy to get pulled into an endless queue of ad-hoc tasks and data quality investigations, but you can consciously make space for creativity.
```

Comment:
I feel like this paragraph and the one preceding would make _great_ introductory paragraphs. You could probably just move them if you wanted to (plus your one-sentence lead-in on the definitions of proficiency vs creativity)